vspec.py

```diff
@@ -203,6 +210,9 @@ def assign_signal_uuids(flat_model):
     return flat_model
 
 
+@deprecation.deprecated(deprecated_in="2.0", removed_in="2.0",
```

Comment:
Can we talk about the intention/usage of the deprecation library?  
Is it for features that may be added/removed from VSS, or from the tooling?

Reply:
I did not want to change the load function from vspec.py as this change would be breaking. I'm also not familiar with all the generators, thus I thought the best way would be to introduce a second similar function (load_tree) and deprecate the old one. I'd recommend to get rid of the load function sooner or later and us the typed model tree to reduce complexity in the generators.